ray-operator/test/sampleyaml/support.go

```diff
@@ -101,3 +101,30 @@ func SubmitJobsToAllPods(t Test, rayCluster *rayv1.RayCluster) func(Gomega) {
 		}
 	}
 }
+
+func GetApps(rayService *rayv1.RayService) map[string]rayv1.AppStatus {
```

Comment:
This function is only called within this file, so it doesn’t need to be public.

```suggestion
func getApps(rayService *rayv1.RayService) map[string]rayv1.AppStatus {
```